src/main/java/teammates/ui/controller/PageData.java

```diff
@@ -685,6 +686,8 @@ public static String getInstructorHoverMessageForFeedbackSession(FeedbackSession
     
     /**
      * Returns the links of actions available for a specific session
+     * @param hasActions
+     *         Indicates whether or not there are possible actions in the first place
```

Comment:
Ok, I saw it, it is generally not a good idea to catch an Exception and not do anything about it though. It should at least have explained why we can ignore the Exception if we are intending to ignore it. "//nothing" just allows it to fail silently and can be hard to debug.

Perhaps, other than returning an empty Action, we can add a logging of some sorts to indicate the failure and where to start finding it's roots etc.

Or perhaps if we can be 100% sure that we can ignore the Exception thrown, we can instead add a comment explaining why and we will no longer need the hasAction boolean.